cmake/mcuboot.cmake

```diff
@@ -103,6 +103,18 @@ if(CONFIG_BOOTLOADER_MCUBOOT)
   set(app_sign_depends
     $<IF:${sign_merged},${merged_hex_file_depends},zephyr_final>)
 
+  if (NOT DEFINED CONFIG_BOOT_SIGNATURE_KEY_FILE)
+    include(${CMAKE_BINARY_DIR}/mcuboot/shared_vars.cmake)
+    set(CONFIG_BOOT_SIGNATURE_KEY_FILE ${mcuboot_SIGNATURE_KEY_FILE})
+    message("
+    ---------------------------------------------------------
+    INFO: CONFIG_BOOT_SIGNATURE_KEY_FILE set from child image to:
+    ${CONFIG_BOOT_SIGNATURE_KEY_FILE}
+    ---------------------------------------------------------
+    \n"
+  )
```

Comment:
I would drop this message, since we have traceability of the origin of the configuration through the `shared_vars.cmake` file. IMO we already have enough warnings going on about the MCUBoot signature key file :D

```suggestion
```